apps/web/.browserslistrc

```diff
@@ -1,4 +1,4 @@
 # https://browserslist.dev/?q=PiAxJSwgbGFzdCAyIHZlcnNpb25zLCBub3QgaWUgPD0gMTE%3D
 > 1%
-last 2 versions
+last 2 versions, Chrome > 105, Edge > 105,
```

Comment:
this should have each declaration in a separate line instead of comma separated - my bad:
> 1%
last 2 version
Chrome > 105
Edge > 105